### PR TITLE
Allow location of CREAM monitoring configuration file to be changed

### LINKED ISSUE #199: template personality/cream_ce/templates/cream-config.templ
JOB_SUBMISSION_MANAGER_SCRIPT_PATH should take /etc/glite-ce-cream-ce-utils/glite_cream_load_monitor.conf instead of  /etc/glite-ce-cream-utils/glite_cream_load_monitor.conf, because of the configuration in personality/cream_ce/monitor.pan which is using hard-coded path to  /etc/glite-ce-cream-ce-utils/glite_cream_load_monitor.conf

### PR DESCRIPTION
**Pull request by jrha**
Not tested, but should fix #199.

### CODE REVIEW

**Comment by Pansanel** [comment]
Test at IN2P3-IRES. Does not change the actual configuration. Perfect!

File: personality/cream_ce/monitor.pan [hunk 1]
```diff
@@ -43,22 +43,24 @@ type cream_monitoring = {
 
 bind '/software/components/metaconfig/services/{${CREAM_MONITORING_CONF_FILE}}/contents' = cream_monitoring;
```
Comment (stdweird) [inline personality/cream_ce/monitor.pan]: wait, this works? never seen that before...

Comment (Pansanel) [reply personality/cream_ce/monitor.pan]: Yes, it works :+1:

Comment (stdweird) [reply personality/cream_ce/monitor.pan]: heh, i didn't know that. in that case it's really a pity that prefix doesn't support it also.

**Pull request merged**